gtp2ogs.js

```diff
@@ -8,8 +8,12 @@ process.title = 'gtp2ogs';
 
 // Do this before importing anything else in case the other modules use config.
 const io = require('socket.io-client');
+
+const { getArgv } = require('./getArgv');
+const argv = getArgv();
+
 const config = require('./config');
-config.updateFromArgv();
```

Comment:
I'm thinking this should be passed to the constructor of the config class, but i think that's too much of a refactor right now.

Reply:
no it may be a good opportunity to do it actually, but i dont know if we pass argv in the constructor and we do `this.argv = argv`, then would argv be exported in config.argv ? (which would be uneeded)